modules/osd-persistent-storage-csi-efs-sts.adoc

```diff
@@ -124,27 +125,23 @@ Save the output. You will use it in the next steps.
 +
 [source,terminal]
 ----
-$ aws iam create-policy \
+POLICY_ARN=$(aws iam create-policy \
   --policy-name "<your_rosa_cluster_name>-rosa-efs-csi" \
   --policy-document file://<your_policy_file_name>.json \
-  --query 'Policy.Arn' --output text) || \
-  POLICY=$(aws iam list-policies \
-  --query 'Policies[?PolicyName==`rosa-efs-csi`].Arn' \
-  --output text
+  --query 'Policy.Arn' --output text)
+echo $POLICY_ARN
 ----
 +
-Save the output. You will use it in the next steps.
 
 .. Attach the IAM policy to the IAM role:
 +
 [source,terminal]
 ----
-$ aws iam attach-role-policy \
-     --role-name "<your_rosa_cluster_name>-aws-efs-csi-operator" \
-     --policy-arn <policy_ARN> <1>
+aws iam attach-role-policy \
```

Comment:
```suggestion
$ aws iam attach-role-policy \
```

I think we need a "$" before a cmd (followed the style). 
Other looks good to me.

Reply:
@ropatil010 Do you also think the same? If we all agree and come to one conclusion, then I can/cannot make changes accordingly. Thank you!

Reply:
@ropatil010  See Line 103 and Line 171

Reply:
Added a "$" before a cmd.